h2o-py/h2o/automl/_estimator.py

```diff
@@ -195,13 +196,15 @@ def __init__(self,
             More models can be trained on an existing AutoML project by specifying the same project name in multiple calls to the AutoML function
             (as long as the same training frame, or a sample, is used in subsequent runs).
         :param exclude_algos: List the algorithms to skip during the model-building phase. 
-            The full list of options is: 
-                ``"DRF"`` (Random Forest and Extremely-Randomized Trees),
-                ``"GLM"``,
-                ``"XGBoost"``,
-                ``"GBM"``,
-                ``"DeepLearning"``,
-                ``"StackedEnsemble"``. 
+            The full list of options is:
```

Comment:
does it render correctly? It needs to end with a double-colon `::` to start a code block:
```suggestion
            The full list of options is::
```

Reply:
Good catch--I was testing out the other list version as well and thought I put it all back to normal. Thanks!

Reply:
also, just noticed this existing use of lists for a similar case in our doc: https://docs.h2o.ai/h2o/latest-stable/h2o-py/docs/frame.html#h2o.H2OFrame.distance

maybe we should use the same format/sphynx syntax everywhere then

Reply:
yes good catch! also slipped in another minor update with the usage examples (thanks to your previous suggestions)